src/components/Beer.vue

```diff
@@ -0,0 +1,136 @@
+<template>
+  <div class="beer">
+    <div class="beer__media">
+      <img class="beer__img" :src="beer.image_url" :alt="beer.name" />
+    </div>
+
+    <div class="beer__wrap">
+      <div class="beer__body">
+        <div class="beer__header">
+          <div class="beer__info">
+            <h2 class="beer__title">{{ beer.name }}</h2>
+            <h3 class="beer__tagline">{{ beer.tagline }}</h3>
+          </div>
+
+          <div class="beer__abv">
+            <div class="beer__abv-label">ALC.VOL</div>
+            <div class="beer__abv-percentage">{{ beer.abv }}%</div>
+          </div>
+        </div>
+
+        <p class="beer__description">{{ beer.description }}</p>
+      </div>
+
+      <div class="beer__pairings">
+        <h4 class="beer__pairings-title">Goes great with</h4>
+
+        <ul class="beer__pairings-list">
+          <li
+            class="beer__pairings-item"
+            v-for="(item , index) in foodPairings"
+            :key="index"
+          >{{ item }}</li>
+        </ul>
+      </div>
+    </div>
+  </div>
+</template>
+
+
+<script>
+export default {
+  name: 'Beer',
+  props: {
+    beer: Object
+  },
+  computed: {
+    foodPairings: function() {
```

Comment:
This is exactly what compute property are intended for.

With ES6, you could remove the function completely:

```
 foodPairings() {
// body
}
```

Reply:
Thanks Aurore, will get this added 👍